src/bin/common.rs

```diff
@@ -302,6 +303,11 @@ pub fn parse_cli() -> Result<CliOptions, CliError> {
         .alias("still_picture")
     )
     // DEBUGGING
+    .arg(
+      Arg::with_name("BENCHMARK")
+        .help("Provide a benchmark report at the end of the encoding")
+        .long("benchmark")
```

Comment:
Would it make sense to make this option hidden from the user?

Reply:
It is a good idea indeed!